guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,19 @@
 [id="katello-headline-features"]
 == Headline Features
 
-[id="katello-upgrade-warnings"]
-== Upgrade Warnings
-
-[id="katello-deprecations"]
-== Deprecations
+* Inter Server Synchronization
+** Complete export has been supported for syncable format export. The hammer content-export option is only available for full content exports. But the content data managed by Katello can be measured easily over 100GB even TB which makes it time consuming and not efficient for complete exports every time.
+** Katello 4.8 adds the ability to perform incremental content exports in syncable format. All three content-export options are available in syncable format.
+  `hammer content-export incremental version --format=syncable`
+  `hammer content-export incremental library --format=syncable`
+  `hammer content-export incremental repository --format=syncable`
+** The export rpm files are generated as hardlinks to the actual file on disk to avoid the 2X space we may need for regular importable exports.
+* Content Views
```

Comment:
And here you could do something like
```suggestion
=== Register hosts to multiple Content Views
```

I'm unsure about the width in the TOC, but you can see in the preview what it looks like, or build it locally:
```sh
cd guides/doc-Release_Notes
make BUILD=katello html browser
```